### PR TITLE
chore: regenerate lock files

### PR DESCRIPTION
**Pull request by connoratrug**
- remove lerna exec yarn install workaround in favour of version task
- version task insures lock file gets (re)gererated

#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Conventional commits (squash if needed)
- [ ] No warnings during install
- [ ] Updated javascript typing

### CODE REVIEW

**Comment by connoratrug** [comment]
[fix master build, extract issue](https://trello.com/c/5vBqWDCi/426-fix-master-build-extract-issue)

File: packages/navigator/config/index.js [hunk 1]
```diff
@@ -12,19 +12,29 @@ module.exports = {
...
-        target: 'http://localhost:8080'
+        target: 'https://master.dev.molgenis.org',
+        changeOrigin: true,
+        secure: false
       },
       '/files': {
-        target: 'http://localhost:8080'
+        target: 'https://master.dev.molgenis.org',
+        changeOrigin: true,
+        secure: false
```
Comment (jvanveen) [inline packages/navigator/config/index.js:37]: You probably wanted to test the navigator against master.dev, but this change is unrelated to fixing the build? If we move the navigator in DE2, this whole file is redundant anyway right?

**Review by jvanveen — APPROVED** [review]
Nice fixes! I think the navigator stuff should probably be apart from this PR, especially because the navigator will be moved in DE2 soon? Anyway, removing flow and fixing some stuff that we encounter now in the navigator may be useful to deal with, without the PR ceremony. I'll leave it up to you whether you want to merge them together.

File: packages/components-library/tests/unit/filters/ChangeFilters.spec.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ describe('ChangeFilters.vue', () => {
   let wrapper: any
   const options = {
     localVue: localVue(),
-    propsData: {
+    propsData: {
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply packages/components-library/tests/unit/filters/ChangeFilters.spec.ts:9]: removed space

File: packages/navigator/build/webpack.base.conf.js [hunk 3]
```diff
@@ -49,7 +49,12 @@ module.exports = {
       {
         test: /\.js$/,
         loader: 'babel-loader',
-        include: [resolve('src'), resolve('test'), resolve('node_modules/webpack-dev-server/client')]
+        include: [
+          resolve('src'), resolve('test'),
+          resolve('node_modules/webpack-dev-server/client'),
+          resolve('node_modules/testdouble'),
+          resolve('node_modules/theredoc')
+        ]
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply packages/navigator/build/webpack.base.conf.js:57]: is needed to have navigator build with updated modules; testdouble and theredoc now need to be transpiled

File: packages/navigator/config/index.js (hunk 1 above)
Comment (connoratrug) [reply packages/navigator/config/index.js:37]: this pr does not move the navigator into de DE2
testing agains the master.dev is the right default here imo as de navigator needs data to test/dev
